securedrop/tests/test_unit_journalist.py

```diff
@@ -7,15 +7,17 @@
 import mock
 
 from flask_testing import TestCase
-from flask import url_for, escape, g
+from flask import url_for, escape
 
 import crypto_util
 import journalist
 import common
-from db import db_session, Source, Journalist, InvalidPasswordLength
+from db import (db_session, Source, Journalist, InvalidPasswordLength,
+                Submission)
 
 # Set environment variable so config.py uses a test environment
 os.environ['SECUREDROP_ENV'] = 'test'
+import config
```

Comment:
Currently, the `securedrop/config.py` modules reads the environment variable `SECUREDROP_ENV`, and sets the configuration differently based on its value. We'd like to deprecate the reading/ setting of environment variables in our code entirely, and instead be read from static INI files.

Although I'm still considering how to do this, I believe that we need to create a separate module for each INI file, for each environment setting. This will involve breaking up `securedrop/manage.py`. Each new module will read only a single INI file. All functionality it provides will involve calls to generators of other modules with initialization arguments, so that no other modules besides these are actually reading from a configuration file.